src/client.ts

```diff
@@ -338,8 +343,8 @@ export class Client extends EventEmitter {
    * @param contractAddress Contract address returned by deploy.
    * @return Uint8Array The corresponding contract code
    */
-  async getCodeAsync(contractAddress: Address): Promise<Uint8Array | null> {
-    const result = await this._readClient.sendAsync<string>('getcode', {
+  async getEVMCodeAsync(contractAddress: Address): Promise<Uint8Array | null> {
```

Comment:
Should be `getEvmCodeAsync`, and similarly for the rest of the new API. We already have `EvmTxReceipt`, and `EvmContract`, so lets stick with the camel case naming.